test/e2e/support/focus-commands.js

```diff
@@ -0,0 +1,34 @@
+Cypress.Commands.add( 'auditBlockFocus', ( blockType ) => {
```

Comment:
Do you think this is a global command or would it be better if we just loop over an array of blocks and do this instead in the test directly? My thinking is that this is very specific to this test and no something we want to share across different tests.

Reply:
It was originally just a forEach in the test. Then, I spit it out into separate tests thinking that I'd use different files for each one. Then I decided to just use one file. Now, it should definitely just be in that one file.